pkg/pool-utils/contracts/BasePoolAuthentication.sol

```diff
@@ -10,6 +10,14 @@ import { Authentication } from "@balancer-labs/v3-solidity-utils/contracts/helpe
 abstract contract BasePoolAuthentication is Authentication {
     IVault private immutable _vault;
 
+    /**
+     * @dev Allow only the swapFeeManager or an authenticated user to call the function.
+     */
+    modifier onlySwapFeeManagerOrAuthentication(address pool) {
```

Comment:
Same as in the other PR, maybe `onlySwapFeeManagerOrAuthenticate` / `onlySwapFeeManagerXorAuthenticate`